ajax/libs/awesome-bootstrap-checkbox/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "awesome-bootstrap-checkbox",
   "filename": "awesome-bootstrap-checkbox.min.css",
   "version": "1.0.0",
-  "description": "Font Awesome Bootstrap Checkboxes & Radios. Pure css way to make inputs look prettier.",
-  "homepage": "http://awesome-bootstrap-checkbox.okendoken.com/demo/index.html",
+  "description": "Font Awesome Bootstrap Checkboxes & Radios. Pure CSS way to make inputs look prettier.",
```

Comment:
It's nice. 
I think you can send a PR to update the upstream description https://github.com/flatlogic/awesome-bootstrap-checkbox/blob/master/package.json#L4
And let this description field as the same with upstream.
Thank you.

Reply:
Okay cool!